src/ofxSyphonServer.mm

```diff
@@ -114,4 +114,70 @@
     [(SyphonServer *)mSyphon publishFrameTexture:id textureTarget:target imageRegion:NSMakeRect(0, 0, width, height) textureDimensions:NSMakeSize(width, height) flipped:!isFlipped];
     [pool drain];
     
-}
\ No newline at end of file
+}
+
+bool ofxSyphonServer::tryToBindForSize(int w,int h){
+  bool res;
+  @autoreleasepool{
+
+    if (!mSyphon)
+    {
+      mSyphon = [[SyphonServer alloc] initWithName:@"Untitled" context:CGLGetCurrentContext() options:nil];
+    }
+
+    res = [(SyphonServer *)mSyphon bindToDrawFrameOfSize:NSMakeSize( w,h)];
+    if(res){
+      ofPushMatrix();
+      ofPushView();
+      ofViewport(0,0,w,h,false);
+      ofSetupScreenOrtho(w,h);
+
+    }
+  }
+  
+		return res;
+}
+
+void ofxSyphonServer::unbindAndPublish(){
+  if (mSyphon){
+    @autoreleasepool{
+      [(SyphonServer *)mSyphon unbindAndPublish];
+
+      updateCurrentTexture();
+      ofPopView();
+      ofPopMatrix();
+    }
+  }
+}
+
+ofTexture & ofxSyphonServer::getCurrentTexture(){
+  return mTex;
+}
+void ofxSyphonServer::updateCurrentTexture(){
+
+  if(!mSyphon) return ;
+  @autoreleasepool{
+    SyphonImage * Img =[(SyphonServer *)mSyphon newFrameImage] ;
```

Comment:
This is a leak - ````newFrameImage```` returns the image retained, and it needs to live for the lifetime of ````mTex```` (or until the next call to ````updateCurrentTexture()```` but then be released